### PR TITLE
added loshchilov2017 2 draft

### PR DESCRIPTION
**Pull request by TongruiLi**

### CODE REVIEW

**Comment by TongruiLi** [comment]
@DartingMelody I can't seem to add you (or anyone) as a reviewer, hence I will have to notify you by the comment here. @zhaoyue-zephyrus if you can provide a fix that will be great!

**Comment by zhaoyue-zephyrus** [comment]
@TongruiLi  I think the reviewer dropped the course so I'll do the review part. Will give some feedback in a few hours.

File: _summaries/loshchilov2017decoupled_2.md [hunk 1]
```diff
@@ -3,17 +3,46 @@ layout: summary
 title: Summary
 paper: {{paper_tag}}
```
Comment (zhaoyue-zephyrus) [inline _summaries/loshchilov2017decoupled_2.md:4]: Please replace `{{page_tag}}` with `loshchilov2016sgdr `. Also the file to edit is `loshchilov2016sgdr_2.md` instead of `loshchilov2017decoupled_2.md`, otherwise we will probably encounter conflicts when merging.

File: _summaries/loshchilov2017decoupled_2.md [hunk 2]
```diff
@@ -3,17 +3,46 @@ layout: summary
...
+
+- The paper incoprporated restart idea from evoluation based and other optimization algorithms to stochastic gradient descent. The origional gradient descent formula is:
+
+$$x_{t+1} = x_t - \eta_t \Delta f_t(x_t)$$
+
+where $$x_{i}$$ represents the parameter at step i, $$\Delta f_i(*)$$ represents the gradient at step i, and $$\eta_i$$ represents the scale of the gradient (otherwise known as the learning rate) at step i.
+
+The paper proposed to modify the learning rate $$\eta$$ by the following:
+
+$$\eta_t = \eta_{min}^i + \frac{1}{2}(\eta_{max}^i - \eta_{min}^i)(1 + cos(\frac{T_{cur}}{T_{i}}\pi))$$
```
Comment (zhaoyue-zephyrus) [inline _summaries/loshchilov2017decoupled_2.md:25]: A figure might be illustrative if any.

File: _summaries/loshchilov2017decoupled_2.md [hunk 3]
```diff
@@ -3,17 +3,46 @@ layout: summary
...
+
+where $$\eta_{t}$$ is constraint between $$[\eta_{min}^i, \eta_{max}^i]$$ and due to the nature of the cosine function, at $$T_{cur} = 0$$ and minimum at $$T_{cur} = T_i$$.  
+
+The paper suggested to make $$T_{cur}$$ discredited values (for example, 0.1, 0.2, 0.3 .. etc) in combination with cosine have a looping behavior. $$T_{i}$$ can be a fixed value, or increased by a factor of $$T_{mult}$$ for better performance.
+
+In addition, the author recommended the training to stop at $$\eta_t = \eta_{min}^i$$ to avoid temporary worsening result due to warm start.
+
 * How well does the paper perform?
+
+The paper presented test errors of state of art models trained on CIFAR-10 and CIFAR-100. When using their methods, the training method improved the CIFAR-10 result from 4.17 to 3.66, and CIFAR-100 result from 20.04 to 18.70 across 5 runs. Even without finetuning, the training method is able to lower the error percent.
```
Comment (zhaoyue-zephyrus) [inline _summaries/loshchilov2017decoupled_2.md:35]: "Even without finetuning, ..." -> What do you mean by "finetuning"? Does it refer to a restart or other? Consider make it more precise.

File: _summaries/loshchilov2017decoupled_2.md [hunk 4]
```diff
@@ -3,17 +3,46 @@ layout: summary
...
+
 * How well does the paper perform?
+
+The paper presented test errors of state of art models trained on CIFAR-10 and CIFAR-100. When using their methods, the training method improved the CIFAR-10 result from 4.17 to 3.66, and CIFAR-100 result from 20.04 to 18.70 across 5 runs. Even without finetuning, the training method is able to lower the error percent. 
+
+In addition, in comparasion to SGD with momentum, SGDR is able to outperform and provide faster convergence across the board.
+
 * What interesting variants are explored?
 
+The paper mostly focuses on the comparasion mentioned above. They also reported ensemble results and found 3.25% and 16.64% test error on CIFAR-10 and CIFAR-100 dataset.
```
Comment (zhaoyue-zephyrus) [inline _summaries/loshchilov2017decoupled_2.md:41]: for "ensemble", I think the way to ensemble is a bit different from the previous methods where the same model is run for multiple times independently. Instead, they use snapshot ensemble. You may briefly discuss it in the method as well.

File: _summaries/loshchilov2017decoupled_2.md [hunk 5]
```diff
@@ -3,17 +3,46 @@ layout: summary
...
+
+They have also run their method on EEG recording classification and demonsrated 1-2% performance improvement.
+
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+* The paper presented SGDR, a novel gradient based optimization technique that allows for faster performance and better convergence on CIFAR-10 and CIFAR-100.
+* Additional variants demonstrated that this method works on models trained on other domains as well, and on ensembles.
+* Incorporating well known method from other related fields may yield better performance.
```
Comment (zhaoyue-zephyrus) [inline _summaries/loshchilov2017decoupled_2.md:48]: What method in particular?

File: _summaries/loshchilov2017decoupled_2.md (hunk 1 above)
Comment (TongruiLi) [reply _summaries/loshchilov2017decoupled_2.md]: Done!

File: _summaries/loshchilov2017decoupled_2.md (hunk 2 above)
Comment (TongruiLi) [reply _summaries/loshchilov2017decoupled_2.md]: added figure below

File: _summaries/loshchilov2017decoupled_2.md (hunk 3 above)
Comment (TongruiLi) [reply _summaries/loshchilov2017decoupled_2.md]: clarified in text.

File: _summaries/loshchilov2017decoupled_2.md (hunk 4 above)
Comment (TongruiLi) [reply _summaries/loshchilov2017decoupled_2.md]: added description

File: _summaries/loshchilov2017decoupled_2.md (hunk 5 above)
Comment (TongruiLi) [reply _summaries/loshchilov2017decoupled_2.md]: added clarification

**Comment by zhaoyue-zephyrus** [comment]
LGTM. merged
